CHANGES-4.3.md

```diff
@@ -22,6 +22,7 @@ File format:
 * Add support for `RSA-PSK-AES256-GCM-SHA384`, `RSA-PSK-AES256-CBC-SHA384`,
  `RSA-PSK-AES128-GCM-SHA256`, `RSA-PSK-AES128-CBC-SHA256` PSK ciphers, and remove `PSK-3DES-EDE-CBC-SHA`,
  `PSK-RC4-SHA` from the default configuration. [#7427]
+* Diagnosis logging for mnesia `wait_for_table`
```

Comment:
```suggestion
* Diagnostic logging for mnesia `wait_for_table`
```

Reply:
also, maybe with more information, e.g. now it logs which nodes the table is pending on etc.

Reply:
what about following? just want to confirm with you avoid wasting CI resources. 

* Diagnostic logging for mnesia `wait_for_table` 
  - prints check points of mnesia internal stats
  - prints check points of per table loading stats
  Help to locate the problem of long table loading time.

Reply:
lgtm